### PR TITLE
One app with webpack

### PR DESCRIPTION
**Pull request by chepkok3**
- Created a table for storing messages and creating 5 different greetings.
- Created an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
- Created a static view that will be the root of the app.
- Created the App component with react-router and render it in the static view.
- Created the Greeting component that will display a greeting. Set it up as a route in the App component.
- Created a store, an action, and a reducer that will connect to the API endpoint to get the random greeting.
- Displayed the random greeting in the Greeting component.

### CODE REVIEW

File: db/migrate/20230613121629_create_greetings.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class CreateGreetings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :greetings do |t|
+
```
Comment (rustam-tolipov) [inline db/migrate/20230613121629_create_greetings.rb]: - [ ] Your Greetings migration is not completed, I think you want to put `:message` as a string. Kinldy accepty my suggestion. 
```suggestion
      t.string :message
```

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_06_13_121629) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
```
Comment (rustam-tolipov) [inline db/schema.rb:16]: - [ ] To fix the above issue and update the schema, kindly delete schema file and run `rails db:reset`!

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,10 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+Greeting.create(message:"Hello, welcome to our website!")
```
Comment (rustam-tolipov) [inline db/seeds.rb:9]: **Kudos for adding seed!**

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @chepkok3,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Well-implemented code_
_Earned a star for adding Greetings in seeds file_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/seeds.rb (hunk 3 above)
Comment (chepkok3) [reply db/seeds.rb:9]: @Rustamxon7 Thanks for highlighting this. I will fix them.

**Review by victorokoroji — APPROVED** [review]
Hi @chepkok3 , 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested changes implemented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**